chorus_lib/examples/hello.rs

```diff
@@ -41,7 +43,10 @@ fn main() {
     let mut handles: Vec<thread::JoinHandle<()>> = Vec::new();
     // Create a transport channel
     // let transport_channel = LocalTransportChannel::<LocationSet!(Bob, Alice)>::new();
```

Comment:
remove this

Reply:
de87e462e4fcd9a040c037e4e4fadb5d3e9d454c